core/src/main/java/org/mapfish/print/servlet/job/HibernateAccounting.java

```diff
@@ -75,15 +77,20 @@ private void insertRecord(final HibernateAccountingEntry tuple) {
         tmpl.execute(
             new TransactionCallbackWithoutResult() {
               @Override
-              protected void doInTransactionWithoutResult(final TransactionStatus status) {
+              protected void doInTransactionWithoutResult(@Nonnull final TransactionStatus status) {
                 final Session currentSession = HibernateAccounting.this.sf.getCurrentSession();
                 currentSession.merge(tuple);
                 currentSession.flush();
                 currentSession.evict(tuple);
               }
             });
       } catch (HibernateException ex) {
-        LOGGER.warn("Cannot save accounting information", ex);
+        String name =
+            MetricRegistry.name(
+                getClass().getSimpleName(), "insertRecordInHibernatrFailedWithException");
```

Comment:
When we read the metrics, we don't want to know that it was an exception!
```suggestion
                getClass().getSimpleName(), "insertRecordInDatabaseFailed");
```